### PR TITLE
Spring application

### PR DESCRIPTION
**Pull request by hejcz**
- simple spring boot application 
- embedded H2 as test db
- the MySQL db is required ( the credentials may be specified in  the `src/main/resources/application.yml` file )
- the integration test was changed to pass
- the angular controller in `src/main/resources/public/hello.js` refers to localhost. It needs to be changed by front-end team

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/pwPIK/PIK-Project/pull/7" target="_blank">ReviewNinja

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+spring:
+    datasource:
+        driverClassName: com.mysql.jdbc.Driver
+        url: jdbc:mysql://localhost:3306/chipdale
+        username: hejcz
+        password: kalafior
```
Comment (ghost) [inline src/main/resources/application.yml:6]: Ciekawe... !resolve

Comment (ghost) [inline src/main/resources/application.yml:6]: Ogólnie to nie wiem co robią niektóre rzeczy, ale wygląda spoko, dam gwiazdkę !

Comment (hejcz) [inline src/main/resources/application.yml:6]: Myślę, że ujdzie. Można wpisać username: username itd. Nie ma to chyba znaczenia skoro to dane do localhosta. !resolved

**Pull request merged**